lib/temporal/workflow/dispatcher.rb

```diff
@@ -39,9 +57,10 @@ def dispatch(target, event_name, args = nil)
 
       def handlers_for(target, event_name)
         handlers[target]
-          .concat(handlers[TARGET_WILDCARD])
-          .select { |event_struct| match?(event_struct, event_name) }
-          .map(&:handler)
+          .merge(handlers[TARGET_WILDCARD]) { raise 'Cannot resolve duplicate dispatcher handler IDs' }
+          .select { |_, event_struct| match?(event_struct, event_name) }
+          .sort
```

Comment:
Kind of surprised that this does the right thing since `handlers[TARGET_WILDCARD]` merged in earlier isn't a DispatchHandler with an associated sequence ID to sort.

Reply:
I see what you mean. Yes, that confused me for sure. Renaming would help.